lib/stormpath-sdk/client.rb

```diff
@@ -20,19 +20,19 @@ class Client
     include Stormpath::Util::Assert
     include Stormpath::Resource::Associations
 
-    attr_reader :data_store, :application
+    attr_reader :data_store, :application, :api_key
 
     def initialize(options)
       base_url = options[:base_url]
       cache_opts = options[:cache] || {}
 
-      api_key = ApiKey(options)
+      @api_key = ApiKey(options)
```

Comment:
I still see we have an api_key property on the client class. This should be removed, unless I'm missing something.